Metadata/Driver/AnnotationDriver.php

```diff
@@ -19,36 +19,47 @@ class AnnotationDriver implements AdvancedDriverInterface
      * @deprecated
      */
     const UPLOADABLE_ANNOTATION = Uploadable::class;
-
+    
```

Comment:
This is bad. Please refer to [StyleCI issues](https://github.styleci.io/analyses/zGAaOl) and fix

Reply:
I guess that's the default behaviour in PHPStorm as I don't remember configuring that
I'll change that as soon as I can.